docker/Dockerfile.cloud-agent

```diff
@@ -1,6 +1,6 @@
-FROM alpine:3.11.3
+FROM alpine:3.13.2
 WORKDIR /home/weave
-RUN apk add --no-cache bash conntrack-tools iproute2 util-linux curl
+RUN apk add --no-cache bash conntrack-tools iproute2 util-linux curl bcc-tools libpcap-dev
```

Comment:
We don't want things relating to 'dev' in the final release container; we have a build container for those.

I backed out this change at https://github.com/weaveworks/scope/commit/5331034059ab9d5ed49afb58f854a8715e87c05e
so we can check the CI run at https://app.circleci.com/pipelines/github/weaveworks/scope/357/workflows/a74f43bf-9696-4bff-bb60-a969f6a02b46

Reply:
@bboreham Thank you for looking at this, tests look good.

Reply:
Ok, do you want to update your branch so I can merge this PR, or shall I merge that branch above?

Reply:
@bboreham Done, I just removed the dev dependencies from my branch.